Client/src/Validation/error.js

```diff
@@ -0,0 +1,25 @@
+const buildErrors = (prev, id, error) => {
+	const updatedErrors = { ...prev };
+	if (error) {
+		updatedErrors[id] = error.details[0].message;
+	} else {
+		delete updatedErrors[id];
+	}
+	return updatedErrors;
+};
+
+const hasValidationErrors = (form, validation, setErrors) => {
+	const { error } = validation.validate(form, {
+		abortEarly: false,
+	});
+	if (error) {
+		const newErrors = {};
+		error.details.forEach((err) => {
+			newErrors[err.path[0]] = err.message;
```

Comment:
Same for this one, always good to fail gracefully if possible

Reply:
fixed in https://github.com/bluewave-labs/bluewave-uptime/pull/986/commits/8ec2f2f16b916eb15cd5d092da768e29b55e6da2, thanks!